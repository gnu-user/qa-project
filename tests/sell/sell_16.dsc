TITLE
    Ticket Volume Input (Edge Case, non integer input, garbage)
DESCRIPTION
    Run the login transaction using an admin account. Initiate the sell 
    transaction. Enter a valid title, and enter valid numerical input (a double 
    with two decimal places between 0.00 and 999.99 inclusive) when prompted to 
    enter a ticket price. Enter a non-numerical value when prompted to enter a 
    ticket volume. Verify that the value is not accepted, and an error occurs.
DEPENDENCIES
    sell_15
