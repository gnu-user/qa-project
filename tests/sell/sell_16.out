Welcome.
Current user accounts file read successfully.
Available tickets file read successfully.
> 
Enter username: 
Username accepted.
> 
Enter the event title: 
Enter the sale price in dollars: 
Enter the number of tickets for sale: 
Invalid number of tickets, volume must be an integer value (e.g. 25).
> 